spec/integration/activate_deactivate_spec.rb

```diff
@@ -0,0 +1,60 @@
+RSpec.describe Moya do
+  describe '#activate' do
+    context 'when requesting hale json' do
+      include_context 'shared drd hale context'
+
+      it 'responds idempotently to an activate call' do
+        # Create deactivated drd
+        req_body = {name: 'deactivated drd', status: 'deactivated', kind: 'standard'}.merge(can_do_hash)
+        response = post(create_url, req_body)
+
+        # Get the activate URL
+        drd = parse_hale(response.body)
+        activate_url = "#{get_transition_uri(drd, "activate")}.hale_json"
+
+        # Activate twice.
+        put activate_url
+        response = put activate_url
+        expect(response.status).to eq(204)
+
+        # Verify
+        response = get hale_url_for("self", drd), can_do_hash
+        expect(parse_hale(response.body).properties['status']).to eq('activated')
+
+        # Destroy our drd
+        delete hale_url_for("delete", drd)
```

Comment:
Should this be in an <code>after</code> block or spec_helper because it's cleanup rather than part of a test